libs/datasets/custom_aggregations.py

```diff
@@ -39,11 +39,11 @@ def calculate_combined_new_york_counties(
     if are_boroughs_zero:
         grouped = non_ny_county.groupby(group).sum()
         for column in grouped.columns:
-            assert sum(grouped[column]) == 0, f"{column} is unexpectedly zero."
+            assert sum(grouped[column]) == 0, f"{column} is unexpectedly not zero."
     else:
         grouped = non_ny_county.groupby(group).sum()
         for column in grouped.columns:
-            assert sum(grouped[column]) != 0, f"{column} is unexpectedly not zero."
+            assert sum(grouped[column]) != 0, f"{column} is unexpectedly zero."
```

Comment:
For assertions, I try to avoid specifying the condition in the message so there isn't risk of desync between the message and *actual* condition. (Hardly a blocker, though, and fixing the existing message is still a nice fix.)

Reply:
agreed - I generally prefer to not do assertions actually, then you can better more contextual python error handling, but probably will just leave this be for now!